exist-core/src/test/xquery/xquery3/empty-direct-constructor.xqm

```diff
@@ -0,0 +1,46 @@
+xquery version "3.1";
+
+module namespace ee="http://exist-db.org/xquery/test/enclosed-expressions";
+
+declare namespace test="http://exist-db.org/xquery/xqsuite";
+
+(:~ Enclosed expressions in element content may be empty in 3.1. Tests borrowed from QT3.
+ :
+ : @see https://github.com/w3c/qt3tests/blob/44252d2496286b0bd0ca1e3b3622f369f0890557/prod/DirElemContent.xml#L1173-L1193
+:)
+
+declare
+    %test:assertEquals("<elem>contentcontent</elem>")
+function ee:K2-DirectConElemContent-26a() {
+    <elem>content{()}content</elem>
```

Comment:
To align with [K2-DirectConElemContent-26a](https://github.com/w3c/qt3tests/blob/44252d2496286b0bd0ca1e3b3622f369f0890557/prod/DirElemContent.xml#L1173-L1182), the empty sequence inside the enclosed expression should be removed.

Reply:
Changed it. Nothing propagates better than errors ;)